zoos/bears.js

```diff
@@ -0,0 +1,127 @@
+const router = require('express').Router();
+const knex = require('knex'); 
+
+const knexConfig = {
+    client: 'sqlite3',
+    useNullAsDefault: true,
+    connection: {
+      filename: './data/lambda.sqlite3',
+    },
+    // debug: true,
+};
+
+const bearDB = knex(knexConfig);
+
+
+// Get all
+router.get('/', (req, res) => {
+    bearDB('bears')
+        .then(names => {
+            if (names.length < 1) {
+                return res.status(404).json({ message: "No bears found" });    
+            }
```

Comment:
not a bad thing to account for by any means, but it could be argued that the developer communicating to your backend would be able to interpret an empty array being returned as a sign that the dataset is currently empty. The message you send back is informative though, so it could go either way.